src/ioc/ioc.react.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import { Container, interfaces } from 'inversify';
+import { useMemoOne } from 'use-memo-one';
+
+const InversifyContext = React.createContext<{ container: Container | null }>({ container: null });
+
+type Props = {
+    container: Container;
+};
+
+export const IoCProvider: React.FC<Props> = (props) => {
+    return (
+        <InversifyContext.Provider value={{ container: props.container }}>
+            {props.children}
+        </InversifyContext.Provider>
+    );
+};
+
+export function useInjection<T>(identifier: interfaces.ServiceIdentifier<T>): T {
```

Comment:
Why here is the ordinary function instead of arrow? You can do the same using arrow function.